resources/vep/plugins/ncER.pm

```diff
@@ -0,0 +1,92 @@
+package ncER;
+
+use strict;
+use warnings;
+
+use Bio::EnsEMBL::Variation::Utils::BaseVepTabixPlugin;
+use base qw(Bio::EnsEMBL::Variation::Utils::BaseVepTabixPlugin);
+
+=head1 NAME
+ ncER
+=head1 SYNOPSIS
+ mv ncER.pm ~/.vep/Plugins
+ ./vep -i variations.vcf --plugin ncER,/FULL_PATH_TO_ncER_file
+=head1 DESCRIPTION
+ Plugin to annotate ncER(https://www.nature.com/articles/s41467-019-13212-3) scores.
+ These scores indicate the likelyhood of a location being essential in terms of regulation.
+ ncER file to be used is the tabixed version provided by GREEN-VARAN (https://github.com/edg1983/GREEN-VARAN) on Zenodo: https://zenodo.org/records/5636163
+ For insertions the anchor based is used to obtain the scores, for larger variants the scores at the start en end position are compared and the highest is used.
```

Comment:
`bed` is 0-based, `vcf` is 1-based: probably requires adjustment in the plugin?

Reply:
fixed